frontend/src/pages/register/form.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react'
+import { Form } from '../../components'
+import { Title } from '../../components/text/text-styles'
+
+const RegisterForm = () => {
+  const [email, setEmail] = useState<string>('')
+  const [password, setPassword] = useState<string>('')
+  const [confirmPassword, setConfirmPassword] = useState<string>('')
+  const [firstName, setFirstName] = useState<string>('')
+  const [lastName, setLastName] = useState<string>('')
+
+  const emailChangeHandler = (event: React.FormEvent<HTMLInputElement>) => {
```

Comment:
This is the code duplication I was talking about that can be avoided with some of the patterns for the form validation. 

Try to figure it out and call if you get stuck ofc